controllers/storageconsumer/storageconsumer_upgrade_controller.go

```diff
@@ -341,16 +341,42 @@ func (r *StorageConsumerUpgradeReconciler) reconcileStorageConsumer(
 		spec.ResourceNameMappingConfigMap.Name = consumerConfigMapName
 
 		spec.StorageClasses = []ocsv1alpha1.StorageClassSpec{
-			{Name: util.GenerateNameForCephBlockPoolStorageClass(storageCluster)},
-			{Name: util.GenerateNameForCephFilesystemStorageClass(storageCluster)},
+			{
+				CommonClassSpec: ocsv1alpha1.CommonClassSpec{
+					Name: util.GenerateNameForCephBlockPoolStorageClass(storageCluster),
+				},
+			},
+			{
+				CommonClassSpec: ocsv1alpha1.CommonClassSpec{
+					Name: util.GenerateNameForCephFilesystemStorageClass(storageCluster),
+				},
+			},
```

Comment:
Similar to the prev example:
```suggestion
		cephBlockPoolStorageClassSpec := ocsv1alpha1.StorageClassSpec{}
		cephBlockPoolStorageClassSepec.Name = util.GenerateNameForCephBlockPoolStorageClass(storageCluster),
		
		cephFsStorageClassSpec := ocsv1alpha1.StorageClassSpec{}
		cephBlockPoolStorageClassSepec.Name = util.GenerateNameForCephFilesystemStorageClass(storageCluster),
		
		spec.StorageClasses = []ocsv1alpha1.StorageClassSpec{
			cephBlockPoolStorageClassSpec,
			cephFsStorageClassSpec
		}
```